### PR TITLE
Add TF summary logging for validate loss

### PR DESCRIPTION
**Pull request by carlogrisetti**
I know it's ugly but this serves the purpose I need. Having few data to work on, training loss and validate loss do not always go hand-in-hand, and this gives me an easy way to have an overview on how validation loss is performing (and infer how likely the model is to be successfully updated in the near future).

### CODE REVIEW

**Comment by carlogrisetti** [comment]
However, I'm starting to suspect that the eval_combined_loss I used is not what I think it is, but just the evaulation of the loss on the training dataset, hence defies my purpose :(

I'm not seeing any real difference in the cases i'm trying to reproduce

**Comment by carlogrisetti** [comment]
Nevermind, i'm adding a feature request since I'm not able to find the real value...
